solr/solr-ref-guide/src/circuit-breakers.adoc

```diff
@@ -107,3 +104,14 @@ circuit breakers checked for a single request can cause a performance overhead.
 
 In addition, it is a good practice to exponentially back off while retrying requests on a busy node.
 
+== Adding New Circuit Breakers
+To add new circuit breakers, add the new circuit breaker's class and required parameters:
+
+[source,xml]
+----
+<circuitBreaker class="foobar" enabled="true">
+ <bool name="enabled">true</bool>
+ <int  name="threshold">75</int>
+</circuitBreaker>
```

Comment:
```suggestion
<circuitBreaker class="foobar" enabled="true">
 <int  name="threshold">75</int>
</circuitBreaker>
```

I wonder if the `enabled` flag could be removed in favour of the `enabled` attribute?